changelog.d/5177.bugfix

```diff
@@ -0,0 +1 @@
+Fix 3pid guest invites.
```

Comment:
actually: could we have a better description of what this is fixing? It looks like it only has an effect if `invite_3pid_guest` is enabled? What does that setting actually do?